### PR TITLE
Camel 14020: Migrate getOut to getMessage in tests.

### PR DESCRIPTION
**Pull request by djoleB**

### CODE REVIEW

File: components/camel-beanstalk/src/main/java/org/apache/camel/component/beanstalk/BeanstalkEndpoint.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
     final ConnectionSettings conn;
 
     @UriPath(description = "Connection settings host:port/tube")
-    private String connectionSettings;
+    private final String connectionSettings;
```
Comment (oscerd) [inline components/camel-beanstalk/src/main/java/org/apache/camel/component/beanstalk/BeanstalkEndpoint.java]: Is there a specific reason?

Comment (djoleB) [reply components/camel-beanstalk/src/main/java/org/apache/camel/component/beanstalk/BeanstalkEndpoint.java]: This needs to be reverted. It's my mistake when i was going through tests.

**Comment by onderson** [comment]
please squash to better reviews..

**Comment by oscerd** [comment]
@davsclaus do you agree on merging this?

**Comment by davsclaus** [comment]
Yeah its a lot of work gone waste if we dont get it merged soonish. Would been good if it was only getOut fixes and not also other changes.

**Comment by oscerd** [comment]
I'll try to merge and build to find eventually problems.

**Comment by oscerd** [comment]
There are some conflicts now. Btw, lets wait for 3.1.0 RC first.

**Comment by djoleB** [comment]
@oscerd @davclaus sorry for late responses i couldn't find time to fix all of this. Do you want me to update branch? So we can resolve those conflicts?

**Comment by oscerd** [comment]
Yes, if you can rebasing would be great. After 3.1.0 RC I'll merge this one.

**Comment by djoleB** [comment]
@oscerd Rebase complete.

**Comment by oscerd** [comment]
Merged, thanks a lot

**Pull request closed without merging**